hedera-node/hedera-app-spi/src/main/java/com/hedera/node/app/spi/config/ConfigProvider.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.node.app.spi.config;
```

Comment:
I this the right module? I would have expected this in `hedera-app`.

Reply:
Will services inject it? I do not know at the moment.

Reply:
Services will probably not have direct access. A `Configuration` will be part of the `XyzContext`, that is passed to the handlers

Reply:
in that case we can move it to app later. I assume we will have a look at the api anyway int future to check if all is really needed in the module.